### PR TITLE
UP-4661: Upgrade 8 portlets and several overlay files

### PR DESCRIPTION
**Pull request by bjagg**

### CODE REVIEW

**Comment by bjagg** [comment]
https://issues.jasig.org/browse/UP-4661

File: uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties [hunk 1]
```diff
@@ -18,12 +18,40 @@
...
+remedyService.baseUrl=${configuration.remedyService.baseUrl}
+
+
+remedyGroupService.username=${configuration.remedyGroupService.username}
+remedyGroupService.password=${configuration.remedyGroupService.password}
+remedyGroupService.locale=${configuration.remedyGroupService.locale}
+remedyGroupService.server=${configuration.remedyGroupService.server}
+remedyGroupService.port=${configuration.remedyGroupService.port}
+remedyGroupService.requestType=${configuration.remedyGroupService.requestType}
+remedyGroupService.baseUrl=${configuration.remedyGroupService.baseUrl}
```
Comment (jameswennmacher) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties]: What is remedy and finance?  I'm not familiar with these.  Are they something that should be part of the overlay?

File: uportal-portlets-overlay/NotificationPortlet/src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+   +-->
+  <root level="${environment.build.log.rootLevel}">
+    <appender-ref ref="LOG" />
+  </root>
+
+  <!-- 
+   | Turn up logging for portlet specific package
+   +-->
+    <!--
+  <logger name="org.jasig.portlet.widget" additivity="false" level="DEBUG">
```
Comment (jameswennmacher) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/logback.xml]: Switch package to Notification

File: uportal-portlets-overlay/WebProxyPortlet/src/main/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+   +-->
+  <root level="${environment.build.log.rootLevel}">
+    <appender-ref ref="LOG" />
+  </root>
+
+  <!-- 
+   | Turn up logging for portlet specific package
+   +-->
+    <!--
+  <logger name="org.jasig.portlet.widget" additivity="false" level="DEBUG">
```
Comment (jameswennmacher) [inline uportal-portlets-overlay/WebProxyPortlet/src/main/resources/logback.xml]: Switch package from widget to proxy

**Comment by jameswennmacher** [comment]
+1

File: uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties (hunk 1 above)
Comment (bjagg) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties]: They are part of the base file. I am actually unfamiliar with these entries.

File: uportal-portlets-overlay/NotificationPortlet/src/main/resources/logback.xml (hunk 2 above)
Comment (bjagg) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/logback.xml]: I missed "Also adjust the portlet package name for the DEBUG log statement that's commented out."

File: uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties (hunk 1 above)
Comment (drewwills) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties]: Guys these properties are a part of some school's implementation of Notification.  They've crept into Apereo versions of this files accidentally.  Please just remove them.  ("$ grep -R "emedy" *" in an up-to-date clone of Notification turns up hits only in this file.)

Comment (bjagg) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties]: Will do. Will create a Jira to remove from portlet project as well.

Comment (bjagg) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties]: What about finance?

File: uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties [hunk 4]
```diff
@@ -18,12 +18,40 @@
...
+
+restApi.enabled=false
+
+studentSuccessPlanService.clientId=SETME
+studentSuccessPlanService.clientSecret=SETME
+studentSuccessPlanService.authenticationUrl=/api/1/oauth2/token
+studentSuccessPlanService.sspProtocol=http
+studentSuccessPlanService.sspHost=localhost
+studentSuccessPlanService.sspPort=80
+studentSuccessPlanService.sspContext=ssp
```
Comment (bjagg) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties]: Should SSP stay as well?

Comment (drewwills) [inline uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties]: I think that one is included with the Notification distribution, so I would think "Yes."

**Pull request merged**